### PR TITLE
(doc) MNGSITE-353: Document maven.repo.local property

### PR DESCRIPTION
**Pull request by AElMehdi**
Following this checklist to help us incorporate your 
contribution quickly and easily:

 - [x] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/MNG) filed 
       for the change (usually before you start working on it).  Trivial changes like typos do not 
       require a JIRA issue.  Your pull request should address just this issue, without 
       pulling in other changes.
 - [x] Each commit in the pull request should have a meaningful subject line and body.
 - [x] Format the pull request title like `[MNG-XXX] - Fixes bug in ApproximateQuantiles`,
       where you replace `MNG-XXX` with the appropriate JIRA issue. Best practice
       is to use the JIRA issue title in the pull request title and in the first line of the 
       commit message.
 - [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [x] Run `mvn clean verify` to make sure basic checks pass. A more thorough check will 
       be performed on your pull request automatically.
 - [ ] You have run the [Core IT][core-its] successfully.

If your pull request is about ~20 lines of code you don't need to sign an
[Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf) if you are unsure
please ask on the developers list.

To make clear that you license your contribution under 
the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)
you have to acknowledge this by using the following check-box.

 - [x] I hereby declare this contribution to be licenced under the [Apache License Version 2.0, January 2004](http://www.apache.org/licenses/LICENSE-2.0)

 - [ ] In any other case, please file an [Apache Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

[core-its]: https://maven.apache.org/core-its/core-it-suite/

### CODE REVIEW

**Comment by AElMehdi** [comment]
@hboutemy,
As discussed before I changed the documentation of the `org.apache.maven.model.interpolation.StringSearchModelInterpolator` based on what it existed in the [previous maven version](https://maven.apache.org/archives/maven-1.x/reference/properties.html#Maven_Configuration_Properties). 
I tried to figure out from the code if it still existed/used which apparently is, since it's referenced in some tests/classes!
Thanks.

File: maven-model-builder/src/site/apt/index.apt [hunk 1]
```diff
@@ -184,6 +184,8 @@ Maven Model Builder
 *----+------+------+
 | <<<maven.build.version>>> | The full build version of the current Maven execution <(since 3.0.4)>. For example, "<<<Apache Maven 3.2.2 (r01de14724cdef164cd33c7c8c2fe155faf9602da; 2013-02-19T14:51:28+01:00)>>>". | <<<$\{maven.build.version\}>>> |
 *----+------+------+
+| <<<maven.repo.local>>> | The repository on the local machine Maven should use to store downloaded artifacts (jars etc). | <<<$\{${user.home}/.m2/repository\}>>> |
```
Comment (michael-o) [inline maven-model-builder/src/site/apt/index.apt]: Something is wrong here: `<<<$\{${user.home}/.m2/repository\}>>>`. Too many braces and dollars.

Comment (AElMehdi) [reply maven-model-builder/src/site/apt/index.apt]: I removed the redundant ones.

**Comment by jira-importer** [comment]
Resolve #6464

**Pull request closed without merging**